### DISCUSSION TITLE
why sail operator not use istiooperator cr resource?

### DISCUSSION THREAD

**Discussion by huanghuangzym** [istio-ecosystem/sail-operator#308] [General]
as istio depreate istio operator
we want to change used sail operator
but  sail operatror  used its self api. 
why not used istiooperator api to install istio?

**Comment by dgn** [comment]
hi @huanghuangzym, we decided against using the IstioOperator API as its future is uncertain- with the upstream operator going away, there's no need for it anymore. Apart from that, it's been incomplete and stalled for a while, with most options only available through the helm values interface, so we went ahead and used the helm APIs directly. If you look at our API, you'll find that it's 98% Istio's helm values API
